deb/templates/deploy.sh

```diff
@@ -0,0 +1,49 @@
+#
+# GRAKN.AI - THE KNOWLEDGE GRAPH
+# Copyright (C) 2018 Grakn Labs Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+#!/usr/bin/env bash
+
+set -ex
+
+if [[ $# -ne 3 ]]; then
+    echo "Should pass <test> <deb-username> <deb-password> as arguments"
+    exit 1
+fi
+
+DEB_REPO_TYPE="$1"
+DEB_USERNAME="$2"
+DEB_PASSWORD="$3"
+
+if [[ "$DEB_REPO_TYPE" != "test" ]]; then
```

Comment:
Let's add
```
TODO: add support deploying to a non-test repository
```

for this one and `deploy_rpm`

Reply:
@lolski I don't think it should be in code. The reason I think so is that I don't know whether mainline repos use the similar procedure ("upload *.deb file by curl") and therefore, the `TODO` item won't be actionable. we have graknlabs/deployment#29 to remind us.

Reply:
I see, let's keep it as is for now then.